client/package.json

```diff
@@ -8,35 +8,40 @@
   "scripts": {
     "build:test": "webpack --config webpack.config.js",
     "build:production": "NODE_ENV=production webpack --config webpack.config.js",
-    "build:development": "webpack -w --config webpack.config.js",
-    "watch": "webpack-dev-server --progress --devtool source-map"
+    "build:development": "webpack -w --config webpack.config.js"
   },
-  "cacheDirectories": ["node_modules", "client/node_modules"],
-  "dependencies": {
+  "cacheDirectories": [
+    "node_modules",
+    "client/node_modules"
+  ],
+  "devDependencies": {
     "babel": "^6.5.2",
     "babel-cli": "^6.6.5",
     "babel-core": "^6.7.4",
+    "babel-eslint": "^6.0.4",
     "babel-loader": "^6.2.4",
-    "babel-runtime": "^6.6.1",
     "babel-polyfill": "^6.7.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
+    "babel-runtime": "^6.6.1",
+    "classnames": "^2.2.3",
+    "eslint": "^2.11.1",
+    "eslint-loader": "^1.3.0",
+    "eslint-plugin-react": "^3.16.1",
+    "imports-loader": "^0.6.5",
+    "webpack": "^1.12.14"
+  },
+  "dependencies": {
     "es5-shim": "^4.5.7",
-    "expose-loader": "^0.7.1",
     "immutable": "^3.7.6",
-    "imports-loader": "^0.6.5",
     "mirror-creator": "1.1.0",
-    "react": "^0.14.8 || ^15.0.0",
-    "react-dom": "^0.14.8 || ^15.0.0",
+    "react": "^15.0",
+    "react-dom": "^15.0",
     "react-on-rails": "6.0.0",
     "react-redux": "^4.4.1",
     "redux": "^3.3.1",
     "redux-promise": "^0.5.3",
-    "redux-thunk": "^2.0.1",
-    "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1"
-  },
-  "devDependencies": {
```

Comment:
webpack-dev-server is only useful if you want to run a webserver with the generated code outside of rails for development. This would only serve the js since the HTML is housed in rails which makes this not terribly useful.

imports-loader is moved to devDependancies

expose-loader was not used and I don't see us having a reason to pollute the window object (expose-loader adds things to the window object).